.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** use custom release line format‏

I would like to change the release line format, which it looks like [we can do here](https://github.com/changesets/changesets/blob/main/docs/modifying-changelog-format.md), to reference the PR instead of the commit and to include the author.

Reply:
Sure, perhaps [Svelte's](https://www.npmjs.com/package/@svitejs/changesets-changelog-github-compact) would be satisfactory already.

Example: https://github.com/vercel/edge-runtime/releases/tag/%40edge-runtime%2Fvm%402.1.2